### PR TITLE
added FetchPartitionusers()

### PR DESCRIPTION
**Pull request by gere-symphony**

### CODE REVIEW

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019 Symphony Communication Services, LLC.
```
Comment (bruceskingle) [inline calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java]: 2020

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import com.symphony.oss.models.allegro.canon.facade.ConnectionSettings;
+import com.symphony.oss.models.core.canon.facade.PodAndUserId;
+import com.symphony.oss.models.object.canon.IPageOfUserPermissions;
+import com.symphony.oss.models.object.canon.IUserPermissionsRequest;
+import com.symphony.oss.models.object.canon.facade.IPartition;
+
+/**
+ * An example application which creates a ToDoItem, adding it to a current and absolute sequence.
+ * 
+ * @author Bruce Skingle
```
Comment (bruceskingle) [inline calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java]: your name

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java [hunk 3]
```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright 2019 Symphony Communication Services, LLC.
```
Comment (bruceskingle) [inline calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java]: 2020

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+import com.symphony.oss.models.core.canon.facade.PodAndUserId;
+import com.symphony.oss.models.core.canon.facade.ThreadId;
+import com.symphony.oss.models.object.canon.IFeed;
+import com.symphony.oss.models.object.canon.facade.IPartition;
+import com.symphony.oss.models.object.canon.facade.IStoredApplicationObject;
+
+/**
+ * Retrieve all objects on the given Sequence.
+ * 
+ * @author Bruce Skingle
```
Comment (bruceskingle) [inline calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java]: your name

**Review by bruceskingle — CHANGES_REQUESTED** [review]

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java (hunk 1 above)
Comment (gere-symphony) [reply calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java]: done

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java (hunk 3 above)
Comment (gere-symphony) [reply calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java]: done

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java (hunk 4 above)
Comment (gere-symphony) [reply calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/DeleteFeed.java]: done

File: calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java (hunk 2 above)
Comment (gere-symphony) [reply calendar/src/main/java/com/symphony/s2/allegro/examples/calendar/FetchPartitionUsers.java]: done

**Pull request merged**